src/modules/ui/components/index.js

```diff
@@ -2,6 +2,7 @@ export { default as Container } from './Container';
 export { default as DataCard } from './DataCard';
 export { default as CardDetail } from './CardDetail';
 export { default as NoteForm } from './NoteForm';
-export { default as FilterBar } from '../../filters/components/FilterBar';
+export { default as Loader } from './Loader';
 
+export { default as FilterBar } from '../../filters/components/FilterBar';
```

Comment:
this is some of old mistakes - its imported from different module into the UI module

Reply:
Deleted